testing/runc/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Dan Williams <dan@ma.ssive.co>
+pkgname=runc
+pkgver=1.0.0_rc5
+pkgrel=0
+pkgdesc="CLI tool for spawning and running containers according to the OCI specification"
+url="https://github.com/opencontainers/runc"
+arch="all"
+license="Apache-2.0"
+depends=""
+makedepends="go bash libseccomp-dev linux-headers"
+install=""
+subpackages=""
+source="https://github.com/opencontainers/runc/archive/v${pkgver/_/-}.tar.gz
+	"
+builddir="$srcdir/$pkgname-${pkgver/_/-}"
+
+build() {
+	export AUTO_GOPATH=1
```

Comment:
This seems to be a variable related to building Docker, nothing Go uses itself as far as I can tell.